### PR TITLE
RDoc-1117 Studio > Database > Tasks > Ongoing Tasks > Backup

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/backup-task.markdown [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  1. If _only_ **Incremental Backup** is set, then a **Full Backup** will occur only in the _first_ time that the Task is triggered,  
+     followed by Incremental Backups according to the scheduled time.  
+     The Full Backup that is done the first time will be either a 'Backup' or a 'Snapshot', depending on the type selected.  
+
+  2. Data that is backed up in **Incremental Backup** is _always_ of type 'Backup' - even if the Backup Task Type is 'Snapshot' !  
+     A Snapshot is only  occurring when scheduling 'Full'.  
+<br/>
+* In the example above:  
+  * A Full Backup is scheduled every day at 02:00 AM - and in addition to that -  
+  * An Incremental Backup is scheduled for every 6 hours
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/backup-task.markdown]: we can drop 'for' here?

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/backup-task.markdown [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+![Figure 3. Backup Task Destinations](images/backup-task-3.png "Backup Destinations")
+
+* Select backup destinations and enter your credentials for each.  
+
+* Note: More than one can be selected.  
+<br/>
+  * Local - Set a folder of your choice (any directory that can be accessed from your machine)  
+  * [Amazon S3](https://aws.amazon.com/s3/)  
+  * [Azure](https://azure.microsoft.com/en-us/services/storage/)  
+  * [Aamazon Glacier](https://aws.amazon.com/glacier/)
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/backup-task.markdown]: typo: Amazon

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/backup-task.markdown [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+![Figure 3. Backup Task Destinations](images/backup-task-3.png "Backup Destinations")
+
+* Select backup destinations and enter your credentials for each.  
+
+* Note: More than one can be selected.  
+<br/>
+  * Local - Set a folder of your choice (any directory that can be accessed from your machine)  
+  * [Amazon S3](https://aws.amazon.com/s3/)  
+  * [Azure](https://azure.microsoft.com/en-us/services/storage/)
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/backup-task.markdown]: Microsoft Azure

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/backup-task.markdown [hunk 4]
```diff
@@ -0,0 +1,149 @@
+﻿# Backup Task
+---
+
+{NOTE: }
+
+* Schedule a Backup Task in order to backup your data.  
+  This will allow you to [restore the database](../../../todo-update-me-later) to a previous state from a specific point in time.  
+<br/>
+* This task is Not replicating your data, see more below in [Backup -vs- Replication](...)
```
[the remark this answers is not in the document]
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/backup-task.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/backup-task.markdown [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+![Figure 1. Backup Task Definition](images/backup-task-1.png "Create New Backup Task")
+
+1. **Task Name** (Optional)  
+   * Choose a name of your choice  
+   * If no name is given then RavenDB server will create one for you based on the defined destination  
+<br/>
+2. **Backup Task Type**:  
+   * ***Backup***  
+     * Backed Up Data: The database data in a json format, including documents, indexes (definitions only) & [identities](../../../todo-update-me-later)
```
[the remark this answers is not in the document]
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/backup-task.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/backup-task.markdown (hunk 1 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/backup-task.markdown]: dropped

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/backup-task.markdown (hunk 2 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/backup-task.markdown]: fixed

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/backup-task.markdown (hunk 3 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/backup-task.markdown]: done

**Pull request merged**